pkg/csi/recover/recover.go

```diff
@@ -118,15 +121,24 @@ func NewFuseRecover(kubeClient client.Client, recorder record.EventRecorder, api
 			return nil, errors.Wrap(err, "failed to parse time period")
 		}
 	}
+	var recoverWarningThreshold int
+	if os.Getenv(RecoverWarningThreshold) != "" {
+		if recoverWarningThreshold, err = strconv.Atoi(os.Getenv(RecoverWarningThreshold)); err != nil {
+			return nil, errors.Wrap(err, "got error when parsing recover warning threshold")
+		}
+	} else {
+		recoverWarningThreshold = defaultRecoverWarningThreshold
+	}
```

Comment:
Please replace the logic with `utils.GetIntValueFromEnv`

Reply:
done.